packages/fs-detectors/test/unit.local-file-system-detector.test.ts

```diff
@@ -1,7 +1,9 @@
 import os from 'node:os';
 import fs from 'node:fs/promises';
 import path from 'node:path';
+import net from 'node:net';
 import { LocalFileSystemDetector, DetectorFilesystem } from '../src';
+import { platform } from 'node:process';
```

Comment:
I've never seen the `node:process` import personally. I'd say stick with `process.platform` (the global) for consistency with the rest of the codebase.

```suggestion
```